app/screens/server/header.tsx

```diff
@@ -41,29 +42,47 @@ const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => ({
     },
 }));
 
-const ServerHeader = ({theme}: Props) => {
+const ServerHeader = ({additionalServer, theme}: Props) => {
```

Comment:
Out of the scope, and personal preference, but why not using the theme from the context?

Reply:
oh sorry.. the  `theme` is used as a prop cause the context is not available when you have not added any servers, so we need to know what theme to use. On first launch is based on the light/dark mode of the device.